js/pillbox.js

```diff
@@ -109,7 +117,7 @@
 
 			if (!$target.hasClass('pill')) {
 				$item = $target.parent();
-				if (this.$element.attr('data-readonly') === undefined) {
+				if (typeof this.$element.attr('data-readonly') === 'undefined') {
```

Comment:
We are using the airbnb rules, which require this. `undefined` still [has some problems](http://eslint.org/docs/rules/no-undefined), but, as we were apparently safely doing without the typeof before, I'll happily overwrite the airbnb rules and we can do without it again.